test/e2e/operator.go

```diff
@@ -279,61 +278,6 @@ var _ = Describe("ARO Operator - Conditions", func() {
 	})
 })
 
-var _ = Describe("ARO Operator - MachineSet Controller", func() {
-	Specify("operator should maintain at least two worker replicas", func() {
-		ctx := context.Background()
-
-		instance, err := clients.AROClusters.AroV1alpha1().Clusters().Get(ctx, "cluster", metav1.GetOptions{})
-		Expect(err).NotTo(HaveOccurred())
-
-		if !instance.Spec.OperatorFlags.GetSimpleBoolean(machineset.ControllerEnabled) {
```

Comment:
If we pull this test out, we can remove [this](https://github.com/Azure/ARO-RP/blob/master/test/e2e/scalenodes.go#L70-L72) as well to re-enable the old one.

It's possible that billing e2e will fail if we don't scale at least once: https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/RP-Billing?path=/e2e/testsuites/billingservice/billingservice_test.go&version=GBmaster&line=228&lineEnd=233&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents